test/app/fiber.test.lua

```diff
@@ -387,7 +395,7 @@ fiber.create(function()end):name()
 --
 -- gh-1926
 --
-fiber.create(function() fiber.wakeup(fiber.self()) end)
+f = fiber.create(function() fiber.wakeup(fiber.self()) end)
```

Comment:
Nit: Please assign to `_` in order to suppress output.

Reply:
Done.